src/server/db/models/enums.ts

```diff
@@ -5,8 +5,9 @@ export enum UserRole {
 
 export enum ContestStatus {
   DRAFT = 'DRAFT',
-  APPROVED = 'APPROVED',
+  IN_REVIEW = 'IN_REVIEW',
   PENDING = 'PENDING',
```

Comment:
Added `IN_REVIEW` contest status:
`IN_REVIEW`: judges are reviewing the contest after it's been submitted
`PENDING`: judges have checked the contest details and requested reward's amount to be transferred
`APPROVED`: fully approved contest, funds were received successfully